CONTRIBUTING.md

```diff
@@ -118,7 +163,9 @@ When appropriate, reference issues (via `#` plus number) .
 For python code style, we use [black](https://github.com/psf/black).
 Please use black before committing any code.
 We will not accept code contributions that do not use black.
+If you have set up your development environment using one of the options above, required formatting tools will be installed and run automatically on any modified files before commits using [pre-commit](https://pre-commit.com/).
```

Comment:
```suggestion
If you have set up your development environment using one of the dev container options specified below, the containers will install all required formatting tools, which will run automatically on any modified files before commits (using a tool called [pre-commit](https://pre-commit.com/)).
```

Minor edits, one to incorporate the future change of adding the devcontainers below